web/assets/css/transmission-app.scss

```diff
@@ -523,8 +523,8 @@ $video-image: '../img/film.svg';
     }
 
     .torrent-labels {
-      font-size: x-small;
-      font-weight: normal;
+      font-size: 0.75em;
+      font-weight: bold;
```

Comment:
Please make font weight normal again for labels in full mode

Reply:
Done